### ISSUE TITLE
Increment value is always set to None for TwoUpOneDownStaircase and ThreeUpOneDownStaircase?

### ISSUE THREAD

**Issue by Gilles86** [VU-Cog-Sci/PRF_experiment#1]
https://github.com/VU-Cog-Sci/PRF_experiment/blob/ed4138aa7607377782b4db42fe9eacdc06fc7aca/exp_tools/Staircase.py#L72

**Comment by tknapen** [comment]
I'll have to check this with @omeomi, as he's the last one to update this part of the code, to my knowledge. He had, at some point, discovered a bug in this code, which I created very quickly and never had the time to test. 
As a quick possibility, perhaps we could see if this code is updated in either another branch of this repo, or the one of his that forks this one? In any case, for your experiment we could just use a set of different coherence values without using a staircase

**Comment by ghost** [comment]
Yes, this is one of the bugs that were in this code. I have a working version of Staircase.py here: https://github.com/omeomi/OriColor_attention_fmri/blob/oricolormapper/exp_tools/Staircase.py
This is not a fork anymore but I can put it in the PRF_experiment if you want. You will have to check that it works though, because I changed quite a few things (and added things as well).

**Comment by tknapen** [comment]
Note that @Gilles86 changed the exp_tools to an actual python package, that at some point we'll all want to start using as a basis for experiments I think. We should get together and discuss what should go into this. The same thing goes for hedfpy, but that's another story

**Comment by ghost** [comment]
That is a good idea, it was very clunky to have the same tools in every experiment

**Comment by Gilles86** [comment]
1) The bug is simply that in the constructor of the TwoUpOneDownStaircase, the parameters sent to the base StairCase-__init__ are always the default. E.g., increment_value is always set to 10:

```python
def __init__(self, initial_value, initial_stepsize, nr_reversals = 10, increment_value = None, stepsize_multiplication_on_reversal = 0.75, max_nr_trials = 40 ):
        super(TwoUpOneDownStaircase, self).__init__(initial_value, initial_stepsize, nr_reversals = 10, increment_value = None, stepsize_multiplication_on_reversal = 0.75, max_nr_trials = 40)
```

So it should be something like
```python
def __init__(self, initial_value, initial_stepsize, nr_reversals = 10, increment_value = None, stepsize_multiplication_on_reversal = 0.75, max_nr_trials = 40 ):
        super(TwoUpOneDownStaircase, self).__init__(initial_value, initial_stepsize, nr_reversals=nr_reversals, increment_value=increment_value, stepsize_multiplication_on_reversal = stepsize_multiplication_on_reversal, max_nr_trials = max_trials)
```

2) You can find the Python-module-version of exptools I've been working on here:
https://github.com/VU-Cog-Sci/exptools
